Add warning about IS7 scope incompatibility (#8009)
wso2/docs-apim: en/docs/administer/key-managers/configure-wso2is7-connector.md

!!! info
    This document provides instructions on configuring **WSO2 Identity Server 7** as a Key Manager. If you are using an older version of WSO2 Identity Server, see [Configure WSO2 IS as a Key Manager]({{base_path}}/administer/key-managers/configure-wso2is-connector).

!!! warning

>     **Incompatibility with Scopes**  

Reviewer: ```suggestion
    **Incompatibility with OAuth2 Scopes**  
```

>     WSO2 API Manager 4.3.0 is **not** compatible with the Role model of WSO2 Identity Server 7. Therefore, scopes are not supported when using WSO2 Identity Server 7 as the Key Manager with WSO2 API Manager 4.3.0.

Reviewer: ```suggestion
    WSO2 API Manager 4.3.0 is **not** compatible with the [Role Based Authorization mode](https://is.docs.wso2.com/en/latest/guides/authorization/api-authorization/api-authorization/)l of WSO2 Identity Server 7.0. Therefore, role based access control with scopes are not supported as a first class feature when using WSO2 Identity Server 7 as the Key Manager with WSO2 API Manager 4.3.0.
```

>     It is not recommended to use WSO2 Identity Server 7 as the Key Manager with WSO2 API Manager 4.3.0, for use cases that use scopes to secure APIs. Consider using a [WSO2 Identity Server version older than 7 as a Key Manager]({{base_path}}/administer/key-managers/configure-wso2is-connector) instead.

Reviewer: ```suggestion
    It is not recommended to use WSO2 Identity Server 7 as the Key Manager with WSO2 API Manager 4.3.0, for use cases that use OAuth2 scopes to secure APIs. Consider using a [WSO2 Identity Server version older than 7 as a Key Manager]({{base_path}}/administer/key-managers/configure-wso2is-connector) instead.
```

2. Add following configurations in the `<IS7_HOME>/repository/conf/deployment.toml` file.

    ```toml
    [server]
    # ... other values ...
    legacy_mode = true
    
    [oauth.global_rbac_scope_issuer]
    enable=true
    

>     [oauth.global_scope_validators.role_based_scope_issuer]

Reviewer: Are we not promoting the old way of creating scopes as well ?

    scopes=[]
    ```

> 3. Add the following unique key constraint to the `IDN_OAUTH_CONSUMER_APPS` table of `WSO2IDENTITY_DB`. By default, the `WSO2IDENTITY_DB` database is an H2 database which is located in `<IS7_HOME>/repository/database/WSO2IDENTITY_DB.mv.db`.

Reviewer: Why dont we need this now?
